src/features/DisplayControls/NursingTasks/utils/NursingTasksUtils.js

```diff
@@ -263,22 +263,6 @@ export const isTimeWithinAdministeredWindow = (
   return enteredTimeInEpochSeconds <= timeWithinWindowInEpochSeconds;
 };
 
-export const disableTaskTilePastNextSlotTime = (
```

Comment:
do we need to remove this method altogether? what if we just keep it and not use it? we can rework this, and if required even provide this under some feature toggle?

Reply:
@angshu  I have kept the method as it is and only removed the usage.